packages/pangea-sdk/pangea/services/authn/models.py

```diff
@@ -0,0 +1,223 @@
+# Copyright 2022 Pangea Cyber Corporation
+# Author: Pangea Cyber Corporation
+
+import enum
+from typing import Any, Dict, List, NewType, Optional
+
+from pangea.response import APIRequestModel, APIResponseModel, PangeaResponseResult
+
+Scopes = NewType("Scopes", List[str])
+Profile = NewType("Profile", Dict[str, Any])
+
+
+class PasswordUpdateRequest(APIRequestModel):
+    email: str
+    old_secret: str
+    new_secret: str
+
+
+class PasswordUpdateResult(PangeaResponseResult):
+    # https://dev.pangea.cloud/docs/api/authn#change-a-users-password
+    pass
+
+
+class IDProvider(str, enum.Enum):
+    FACEBOOK = "facebook"
+    GITHUB = "github"
+    GOOGLE = "google"
+    MICROSOFT_ONLINE = "microsoftonline"
```

Comment:
Nit: Can this just be MICROSOFT? The _ONLINE part seems unnecessary.

Reply:
Yes! Fixed!